selene/src/json_output.rs

```diff
@@ -11,6 +11,7 @@ pub enum JsonOutput {
 
 #[derive(Serialize)]
 pub struct JsonDiagnostic {
+    filename: String,

```

Comment:
I think filename makes more sense to be part of Span (or Label). This way in the future it can support primary and secondary labels from different files.

Reply:
Ooh that is a good point